### ISSUE TITLE
Unable to launch app.

### ISSUE THREAD

**Issue by Npepperlinux** [ubuntu/drawing#2]
Related to https://github.com/maoschanz/drawing/issues/612

The following error message is displayed only and cannot be started.

```
/snap/drawing/28/gnome-platform/command-chain/desktop-launch: /snap/drawing/28/usr/bin/drawing: /snap/gnome-42-2204-sdk/current/usr/bin/python3: bad interpreter: No such file or directory
/snap/drawing/28/gnome-platform/command-chain/desktop-launch: line 605: /snap/drawing/28/usr/bin/drawing: Success
```

I think that is because [`#!@PYTHON@`](https://github.com/maoschanz/drawing/blob/9e2cc6c74a1085b012101315fbc821595c7785fa/src/drawing.in#L1C1-L1C11) is replaced by `#!/snap/gnome-42-2204-sdk/current/usr/bin/python3` at the time of build.

**Comment by Npepperlinux** [comment]
I can't launch apps on Ubuntu 24.10.

```
/snap/drawing/39/gnome-platform/command-chain/desktop-launch: line 647: /snap/drawing/43/usr/bin/drawing: cannot execute: required file not found
```

The base in `/snap/drawing/43/snap/snapcraft.yaml` specifies core24, but the fix for #3 does not appear to have been updated.

**Comment by Npepperlinux** [comment]
> I can't launch apps on Ubuntu 24.10.
> 
> ```
> /snap/drawing/39/gnome-platform/command-chain/desktop-launch: line 647: /snap/drawing/43/usr/bin/drawing: cannot execute: required file not found
> ```
> 
> The base in `/snap/drawing/43/snap/snapcraft.yaml` specifies core24, but the fix for [#3](https://github.com/ubuntu/drawing/pull/3) does not appear to have been updated.

Sorry, I see the repository had changed.
